Makefile

```diff
@@ -13,6 +14,9 @@ test_app_instrumentation_debug:
 test_app_instrumentation:
 	./gradlew subprojects\:android-test\:test-app\:instrumentationUi -PinfraVersion=local -Pci=true -PtestBuildType=$(test_build_type) $(log_level)
 
+test_app_debug:
```

Comment:
Bad name.

Reply:
`test_app_instrumentation_debug` is already reserved. How about `test_app_instrumentation_tests_debug`?

Reply:
let rename `test_app_instrumentation_debug` to `test_app_instrumentation_gradle_debug`
New we can name `test_app_instrumentation_android_debug`